### ISSUE TITLE
Double menu bar in LibreOffice Writer for Linux with gtk3 backend, french UI and LanguageTool >= 4.4

### ISSUE THREAD

**Issue by roland65** [languagetool-org/languagetool#1603]
In Linux, when the LanguageTool >= 4.4 is installed as an extension in LibreOffice Writer, with french UI and gtk3 backend, then opening a simple .odt document results in Writer having a double menu bar (see attached screenshot).

Test cases to reproduce and analyze the issue:

1. Install LibreOffice 6.2.4.2 with french UI (important)

2. Install LanguageTool >= 4.4 in LibreOffice Writer. Note that Writer crashes at restart

3. Start Writer with gtk3 backend (and french UI)
SAL_USE_VCLPLUGIN=gtk3 /opt/libreoffice6.2/program/swriter

4. Open the attached 'test_document.odt' simple document => Writer now has a double menu bar (see attached screenshot)

5. Start Writer with french UI and gtk (or gen) backend
SAL_USE_VCLPLUGIN=gtk /opt/libreoffice6.2/program/swriter
or
SAL_USE_VCLPLUGIN=gen /opt/libreoffice6.2/program/swriter

6. Open the attached 'test_document.odt' simple document => Writer has a single menu bar

7. Start Writer with default (US) UI and gtk3 backend
LANG=C SAL_USE_VCLPLUGIN=gtk /opt/libreoffice6.2/program/swriter

8. Open the attached 'test_document.odt' simple document =>  Writer has a single menu bar

So, to summarize,  the double menu bar issue appears only with french UI and gtk3 backend and with LanguageTool >= 4.4. If LanguageTool is not installed or if LanguageTool < 4.4 is installed, then there is no such issue. 

System: Ubuntu Linux 18.04.2 64 bits
LibreOffice 6.4.2.4 downloaded from www.libreoffice.org
LanguageTool 4.4, 4.4.1, 4.5, 4.5.1 oxt extension
Java: openjdk-11-jre:amd64

All tests were done with a fresh LibreOffice profile.

[test.zip](https://github.com/languagetool-org/languagetool/files/3232355/test.zip)
